Libraries/Geolocation/RCTLocationObserver.m

```diff
@@ -135,7 +149,9 @@ - (dispatch_queue_t)methodQueue
 
 - (void)beginLocationUpdatesWithDesiredAccuracy:(CLLocationAccuracy)desiredAccuracy distanceFilter:(CLLocationDistance)distanceFilter
 {
-  [self requestAuthorization];
+  if (!_locationConfiguration.skipPermissionRequests) {
```

Comment:
wondering if we can just check if permissions are already granted instead of a skip flag